bip-0159.mediawiki

```diff
@@ -36,8 +36,6 @@ This BIP proposes two new service bits
 | NODE_NETWORK_LIMITED_HIGH || bit 11 (0x800) || If signaled, the peer <I>MUST</i> be capable of serving at least the last 1152 blocks (~8 days)
```

Comment:
@@ -36,8 +36,6 @@ This BIP proposes two new service bits

 
 | NODE_NETWORK_LIMITED_HIGH || bit 11 (0x800) || If signaled, the peer <I>MUST</i> be capable of serving at least the last 1152 blocks (~8 days) 

 
 | NODE_NETWORK_LIMITED_HIGH || bit 11 (0x800) || If signaled, the peer <I>MUST</i> be capable of serving at least the last 1152 blocks (~8 days) 

 
 
 
 
 
 
 
 
 
 
 
 @@ -36,8 +36,6 @@ This BIP proposes two new service bits

 
 | NODE_NETWORK_LIMITED_HIGH || bit 11 (0x800) || If signaled, the peer <I>MUST</i> be capable of serving at least the last 1152 blocks (~8 days) 

 
 | NODE_NETWORK_LIMITED_HIGH || bit 11 (0x800) || If signaled, the peer <I>MUST</i> be capable of serving at least the last 1152 blocks (~8 days) 

 
 
 
 
 
 
 
 
 
  Show comments  
 
 
View   
2 bip-0159.mediawiki 




 
@@ -36,8 +36,6 @@ This BIP proposes two new service bits

 
 | NODE_NETWORK_LIMITED_HIGH || bit 11 (0x800) || If signaled, the peer <I>MUST</i> be capable of serving at least the last 1152 blocks (~8 days) 
 wait a sec....... look Now thinking about it I set those guys loose on a server rack at my buddies house and you're telling me that he wasn't getting anytjing.as a matter a fact they were even renting hash power from me at the time . do you think that might have anything to do with the whole thing? I forgot all about that, and nobody mentioned anything about the hash rate that was being rented out.......That's probably why the high hash rates now its making sense. Maybe the block chain isn't flawed and its us humans that are fucking up as usual. the computer is fine and some It's probably shaking its head at us just like you at me?????????????????
Bingo, I solved to million $$ flaw that would have been really bad when in reality its human error like always 
Did you get a chance to read this? So it's probably the guys running the rig who got over and I;m a perfect scape goat as you can see.......................................................... Those fuckers almost got away with it too. I was running them on another site as well, cant remember the name of it but I know they were renting hash power. Shit I forgot all about that till now..........

Reply:
Damn, should have proof read that. See the truth comes out..... I HOPE SOMEONE FEELS BAD that they were just going to trick me and play me for a fool.  I wasn't trying to get over on anyone... And the other side?????????
well were not supposed to talk about it..